packages/blade/src/components/ActionList/actionListUtils.ts

```diff
@@ -77,7 +77,21 @@ const getActionListProperties = (
       actionListOptions.push({
         title: child.props.title,
         value: child.props.value,
-        href: child.props.href,
+        onClickTrigger: (value) => {
+          const anchorLink = child.props.href;
+          child.props.onClick?.({
+            name: child.props.value,
+            value: child.props.isSelected ?? value,
+          });
+
+          if (anchorLink) {
+            const target = child.props.target ?? '_self';
+            window.open(anchorLink, target);
+            if (window.top) {
+              window.top.open(anchorLink, target);
+            }
```

Comment:
Is this running on web only? `window` won't work on native

Reply:
yes it only runs when you click item with keyboard down which won't happen in mobile. I've added isReactNative check anyway to avoid breakage in case that happens in some edge case. But ideally `onKeyboardDown` callback itself doesn't exist on native so shouldn't happen.